Test/AbstractBlockServiceTestCase.php

```diff
@@ -97,4 +97,20 @@ protected function assertSettings(array $expected, BlockContextInterface $blockC
 
         $this->assertSame($completeExpectedOptions, $blockSettings);
     }
+
+    /**
+     * NEXT_MAJOR: Remove this method when dropping support for < PHPUnit 5.4.
+     *
+     * @param string $class
+     *
+     * @return \PHPUnit_Framework_MockObject_MockObject
+     */
+    protected function createMock($class)
```

Comment:
> Not sure about the right way.

No need to be ultra-right here, it's code that will disappear, so just do what you feel is simpler, bearing in mind that if we go @osavchenko , we will still have to change the `extends` one way, and the other way around when removing this on next major.